mix.exs

```diff
@@ -22,13 +22,13 @@ defmodule AMQP.Mixfile do
   end
 
   def application do
-    [applications: [:logger, :amqp_client]]
+    [applications: [:logger, :rabbit_common, :amqp_client]]
```

Comment:
any reason you needed this change?

Reply:
I added this in an attempt to get the failing test to work, which failed. Apologies. Let me remove it.

Reply:
done @ono